README.md

```diff
@@ -584,12 +584,20 @@ The implementation and usage of the mentioned `DateTime` to `int` converter is d
 class DateTimeConverter extends TypeConverter<DateTime, int> {
   @override
   DateTime decode(int databaseValue) {
-    return DateTime.fromMillisecondsSinceEpoch(databaseValue);
+    try {
+      return DateTime.fromMillisecondsSinceEpoch(databaseValue);
+    } catch (e) {
+      return null;
+    }
```

Comment:
❓Is it possible to make `DateTime.fromMillisecondsSinceEpoch` throw as long as the given value is not `null`? If not, we could simplify these lines by checking if the `databaseValue` is `null`. If not, we create a `DateTime`. In case it's `null`, we return `null`.